GUI/Util/UpdateGUISettings.cs

```diff
@@ -0,0 +1,253 @@
+using System.Threading.Tasks;
+using TaskLayer;
+
+namespace MetaMorpheusGUI
+{
+    public static class UpdateGUISettings
+    {
+        public static GuiGlobalParams Params { get; internal set; }
+
+        public static bool LoadGUISettings()
+        {
+            bool fileExists= File.Exists(Path.Combine(GlobalVariables.DataDir, @"GUIsettings.toml"));
+            try //catches toml read errors
+            {
+                Params = Toml.ReadFile<GuiGlobalParams>(Path.Combine(GlobalVariables.DataDir, @"GUIsettings.toml"));
+            }
+            catch
+            {
+                Params = new GuiGlobalParams(); //create an instance
+                Toml.WriteFile(Params, Path.Combine(GlobalVariables.DataDir, @"GUIsettings.toml"), MetaMorpheusTask.tomlConfig);
+            }
+
+            if (GlobalVariables.MetaMorpheusVersion.Contains("Not a release version"))
+            {
+                Params.AskAboutUpdating = false;
+            }
+            return fileExists;
+        }
+
+
+        public static bool UseNonSpecificRecommendedSettings()
+        {
+            bool useRecommendedSettings = false;
+            //check with the user to update params
+            if (Params.AskAboutNonSpecificParams )
+            {
+                var results = ProteaseSpecificMsgBox.Show("Use Non-Specific Recommendations?",
+                    "We recommend using the following parameters for non-specific searches:\n" +
+                    "\t-Check 'Non-Specific Search' (Search Task Only)\n" +
+                    "\t-Use '25' for 'Max Peptide Length'.\n\n" +
+                    "Would you like to use these recommended settings?");
+
+                if (results.UseSettings)
+                {
+                    useRecommendedSettings = true;
+                }
+                //else do nothing
+
+                //if they don't want to see this again, save the answer
+                if (!results.AskAgain)
+                {
+                    Params.AskAboutNonSpecificParams = false;
+                    Params.UseNonSpecificParams = results.UseSettings;
+
+                    Toml.WriteFile(Params, Path.Combine(GlobalVariables.DataDir, @"GUIsettings.toml"), MetaMorpheusTask.tomlConfig);
+                }
+            }
+            else if (Params.UseNonSpecificParams) //user didn't want to check in, but wanted to update last time
+            {
+                useRecommendedSettings = true;
+            }
+            //else do nothing
+            return useRecommendedSettings;
+        }
+
+        public static bool UseTopDownRecommendedSettings()
+        {
+            bool useRecommendedSettings = false;
+            //check with the user to update params
+            if (Params.AskAboutTopDownParams )
+            {
+                var results = ProteaseSpecificMsgBox.Show("Use Top-Down Recommendations?",
+                    "We recommend using the following parameters for top-down searches:\n" +
```

Comment:
I also recommend variable mods be disabled for top-down searches - want to add this?

Reply:
Agreed. If GPTMD is done and includes oxidation of M, then variable mods should not be done.

Without GPTMD, using a .fasta database (yeast):
1. Variable Ox
 PSMs: 509
 Time: 41 sec

2. No Variable Mods
 PSMs: 723
 Time: 22 minutes

So it's worth looking for oxidation, but there's a 30 fold increase in search time to do it using a variable search.